### PR TITLE
Move to yarn 2 and pnp for engagement-view

### PR DESCRIPTION
**Pull request by colin-grapl**
### Which issue does this PR correspond to?
https://github.com/grapl-security/issue-tracker/issues/389

### What changes does this PR make to Grapl? Why?
https://next.yarnpkg.com/features/zero-installs

Starts the process to move us to zero-installs. The step after this would be to move us to a workspace.

### How were these changes tested?
So far I've only confirmed that builds still succeed.

### CODE REVIEW

File: src/js/engagement_view/src/tests/engagementView/data/graphVizData.tsx [hunk 1]
```diff
@@ -1,85 +1,76 @@
...
+				dgraph_type: ["Risk"],
+				node_key: "Rare Parent of cmd.exe",
+				analyzer_name: "Rare Parent of cmd.exe",
+				risk_score: 10,
+			},
+		],
+	},
+	{
+		uid: 10013,
+		node_key: "DESKTOP-FVSHABR",
```
Comment (inickles-grapl) [inline src/js/engagement_view/src/tests/engagementView/data/graphVizData.tsx:36]: Changes like these seem irrelevant to this PR title and description. Should this be a in a separate PR?

**Review by inickles-grapl — COMMENTED** [review]
I know this is WIP, but:
1. It looks like there are other changes to this PR than just yarn 2 and pnp for Engagement View
2. Per what you'd said in chat, I thought this would include previously built yarn dependencies, but I don't see them? Were they just not added?

File: src/js/engagement_view/public/index.html [hunk 2]
```diff
@@ -2,11 +2,12 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <link rel="icon" href="./grapl_logo.png" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="Grapl: A Graph Analytics Platform for Detection & Response"
+      content="Web site created using create-react-app"
```
Comment (wimax-grapl) [inline src/js/engagement_view/public/index.html:10]: ??

File: src/js/engagement_view/public/index.html [hunk 3]
```diff
@@ -23,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <link rel="icon" href ="./grapl_logo.png">
+    <link rel="icon" href ="./grapl_logo.png"></icon>
```
Comment (wimax-grapl) [inline src/js/engagement_view/public/index.html:27]: ??

File: src/js/engagement_view/src/tests/engagementView/data/graphVizData.tsx (hunk 1 above)
Comment (colin-grapl) [reply src/js/engagement_view/src/tests/engagementView/data/graphVizData.tsx]: Fixed (2), yeah idk about (1) lemme look into this.
